Doc/reference/lexical_analysis.rst

```diff
@@ -656,7 +656,8 @@ the final value of the whole string.
 
 Top-level format specifiers may include nested replacement fields.
 These nested fields may include their own conversion fields and
-format specifiers, but may not include more deeply-nested replacement fields.
+:ref:`format specifiers <formatspec>`, but may not include more
```

Comment:
great addition, but it would also be good to explicitly state, without having to follow a link, that f-strings use the same format mini-langage as the .format() method. Maybe:

These nested fields may include their own conversion fields and format specifiers, but may not include more deeply-nested replacement fields. The :ref:`format specifier mini-language <formatspec>` is the same as that used by the string .format() method.

maybe put the ref in the seconds sentence?

Reply:
Makes sense :) I'll make another PR later today.